add-ons/argo-workflows/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
```

Comment:
We typically just create a reference to the original helm chart using the `dependencies` key. See others for examples.

Reply:
fixed